mm/ktsan/ktsan.c

```diff
@@ -251,7 +253,7 @@ void ktsan_print_diagnostics(void)
    function, but it requires access to ENTER and LEAVE. */
 void kt_tests_run(void)
 {
-	ENTER(false, false);
+	ENTER(0);
```

Comment:
Maybe add another macro, that would be defined to 0?
It's not really clear what 0 in ENTER(0) means.